app/routes/home-pro.js

```diff
@@ -1,5 +1,12 @@
 import TravisRoute from 'travis/routes/basic';
+import config from 'travis/config/environment';
 
 export default TravisRoute.extend({
-  needsAuth: false
+  needsAuth: false,
+
+  activate: function() {
```

Comment:
I think that a better place to fix this behaviour would be to change our `Location` handler. At the moment we handle home pages with `if(config.pro)` check: https://github.com/travis-ci/travis-web/blob/master/app/utils/location.js#L21-L31

I think that if we avoid logic for homepage for enterprise altogether, it will result in cleaner code, so we could change [this statement](https://github.com/travis-ci/travis-web/blob/1d0fb6468e83f6b99d16f15e1bd2a095e67c6380/app/utils/location.js#L22) to `if(location.pathname === '/' && !config.enterprise)` and [this statement](https://github.com/travis-ci/travis-web/blob/1d0fb6468e83f6b99d16f15e1bd2a095e67c6380/app/utils/location.js#L37) to `if (!config.enterprise && (logicalPath === '/repositories' || logicalPath === '/home' || logicalPath === '/home-pro'))`. With these conditions we would run regular URL handling for enterprise, so no homepage would be displayed initially.